src/core/migrations/0082_mark_safe_article_title.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import re
+from django.conf import settings
+from django.db import migrations
+
+REGEX = re.compile("{{\ ?([a-z\._-]*)article.title\ ?}}")
```

Comment:
This REGEX would miss `{{ author_review.assignment.article.title|safe }}` in the `author_copyedit_deleted` email.

Reply:
> This REGEX would miss `{{ author_review.assignment.article.title|safe }}` in the `author_copyedit_deleted` email.

I thought of this, but since `**.article.title|safe` is already a valid value, I thought it'd be best to not overly-complicate the regex. I can extend it to catch and  remove the `|safe` though

Reply:
Nah it’s fine. You’re right!